### PR TITLE
Fix/long archive time

### PR DESCRIPTION
**Pull request by LukasHromadnik**
Hey. :)

I think I fix the long archive time: https://github.com/thii/FontAwesome.swift/issues/206.

I've changed the `codegen` to generate only an enumeration with an unicode value of the icon as a computed property.

Can you please look at the change? AFAIK the changed code is working, all tests passed successfully, but I wanted to be 100 % sure.

PS: The `codegen` was also generating enum for FontAwesomeBrands but this enum is not used inside the library at all. So I commented out this part, is it alright?

### CODE REVIEW

**Comment by PRCI** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: codegen.swift [hunk 1]
```diff
@@ -81,44 +81,49 @@ public enum FontAwesome: String {
...
 fontAwesomeEnum += """
+        
+        }
+    }
+}
 
-]
-
+"""
+/*
```
Comment (thii) [inline codegen.swift]: Why is this commented out?

Comment (LukasHromadnik) [reply codegen.swift]: I'd like to delete that part. But I don't know why it is there. The generated part is not used anywhere inside the framework.

Comment (thii) [reply codegen.swift]: It was added via #202 and #203.

Comment (thii) [reply codegen.swift]: It's part of the public interfaces and is used by the users of this framework.

Comment (LukasHromadnik) [reply codegen.swift]: Okay, I didn't know that. So in that case I will update codegen to generate this enumeration as well.

**Comment by LukasHromadnik** [comment]
Yes, I have. Currently on my computer (i7, 4 cores, 16 GB RAM) the archive process takes about 50 seconds.

**Comment by thii** [comment]
This has been released in 1.8.0. Thanks again!

**Pull request merged**